enterprise/models.py

```diff
@@ -589,16 +591,18 @@ class Meta(object):
     NOT_SET = 'not_set'
     ENABLED = 'enabled'
     DISABLED = 'disabled'
+    EXTERNALLY_MANAGED = 'externally_managed'
     STATE_CHOICES = (
         (NOT_SET, 'Not set'),
         (ENABLED, 'Enabled'),
         (DISABLED, 'Disabled'),
+        (EXTERNALLY_MANAGED, 'Managed Externally'),
     )
 
     user = models.ForeignKey(EnterpriseCustomerUser, related_name='data_sharing_consent')
 
     state = models.CharField(
-        max_length=8,
+        max_length=20,
```

Comment:
Would we have enough data in PROD at this point for this to be a problem? Though I suppose it might be a good thing to shorten the new state value string down.